package.json

```diff
@@ -1,17 +1,18 @@
 {
   "name": "@digicatapult/inteli-api",
-  "version": "1.21.0",
+  "version": "1.22.0",
   "description": "Insert repo description",
   "main": "app/index.js",
   "scripts": {
     "db:migrate": "npx knex migrate:latest --env test",
     "test": "NODE_ENV=test mocha --config ./test/mocharc.js ./test",
     "test:unit": "NODE_ENV=test mocha --config ./test/mocharc.js ./app",
+    "test:jwt": "NODE_ENV=test AUTH_TYPE=JWT mocha --config ./test/mocharc.js ./test",
     "lint": "eslint .",
     "depcheck": "depcheck",
     "start": "NODE_ENV=production node app/index.js",
     "dev": "NODE_ENV=development nodemon app/index.js | pino-colada",
-    "coverage": "nyc npm run test",
+    "coverage": "LOG_LEVEL=fatal NODE_ENV=development nyc mocha --recursive ./test/integration --timeout 60000 --slow 20000 --exit",
```

Comment:
not sure how best to do a merge as `npm run test` and `npm run test:jwt` require different config for dependencies (set with `AUTH_TYPE: NONE` and `AUTH_TYPE: JWT` respectively).

Could add `IDENTITY_AUTH_TYPE=JWT DSCP_API_AUTH_TYPE=JWT docker compose up -d` to `test:jwt`?

Reply:
added